File: 1-easy/problems/3936-minimum-swaps-to-move-zeros-to-end/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def minimumSwaps(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        return sum(nums[i] != 0 for i in range(len(nums)-nums.count(0), len(nums)))

File: 1-easy/problems/3936-minimum-swaps-to-move-zeros-to-end/solution.ts
function minimumSwaps(nums: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :rtype: int
  """
  return sum(nums[i] != 0 for i in Array.from({length: (nums}, (_, i) => i).length-nums.count(0), (nums).length));
}

export { minimumSwaps };
